canonical/ubuntu.com
# Align remaining 18.04 pages

## Done

- Updated copy on respective 18-04 pages as per docs
- Changed GCP to Google Cloud
- Fixed medium screen alignment 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/18-04/azure
- Check against [doc](https://docs.google.com/document/d/1O7RApX5BNPH13-3j-Wq-31RWz-pdpciNFea7ixL3kJI/edit#)
- View the site locally in your web browser at: http://0.0.0.0:8001/18-04/gcp
- Check against [doc](https://docs.google.com/document/d/1d62l-BdsgXjs0PZ_JuxJ0u_xG7lUNqyFVBN-lCvSt00/edit#)
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4182
